xbundle/__init__.py

```diff
@@ -250,7 +250,12 @@ def import_metadata_from_directory(self, dir):
             policies = etree.Element('policies')
             policies.set('semester',os.path.basename(pdir))
             for fn in glob.glob(join(pdir, '*.json')):
-                x = etree.SubElement(policies,os.path.basename(fn).replace('_','').replace('.json',''))
+                try:
+                    x = etree.SubElement(policies,os.path.basename(fn).replace('_','').replace('.json',''))
```

Comment:
cool, we can leave the context manager out, but add the conditional and I'll merge